scipy/integrate/quadrature.py

```diff
@@ -354,6 +355,10 @@ def simps(y, x=None, dx=1, axis=-1, even='avg'):
 
         'last' : Use Simpson's rule for the last N-2 intervals with a
                trapezoidal rule on the first interval.
+    method : {'composite', 'extended', 'second'}, optional
+        'composite' : Uses composite method of Simpson's rule.
+        'second' : Uses Simpson's 3/8-th rule for calculating integral.
+        'extended' : Uses extended Simpson's rule for calculating integral.
```

Comment:
The methods need to be explained. Ideally, give exact formulas in the Notes section of the docstring.